validphys2/src/validphys/plotoptions/kintransforms.py

```diff
@@ -69,6 +69,31 @@ def xq2map(self, k1:np.array,k2:np.array,k3:np.array,**extra_labels) -> (np.arra
 import numpy as np
 
 
+# TODO
+class identity:
+    """The identity transformation is a transitional operation for the implementation
+    of the new commondata
+
+    It takes the kinematics as they come and leave them unchanged.
+    The final version will either
+        1. Remove the necessitity for the key in the PlotOptions
+        2. Change all "identities" to utilize one of the transformations
+    The selection depends on the final decision on the x-q2 map and the possibilities are
+    either:
+        1. Include the xq2 information as part of the kinematics when no override is provided
+        2. Enforce that all datasets include an override to define how the x-q2 map should be computed.
+    """
+    def __call__(self, k1, k2, k3):
+        return k1, k2, k3
+
+    def new_labels(self, k1, k2, k3):
+        return k1, k2, k3
+
+    def xq2map(self, k1, k2, k3, **extra_labels):
+        # This is going to be a problem
+        return k1, k2
```

Comment:
What exactly is this supposed to do?

Reply:
Making sure that the code doesn't crash. But the results are wrong (the xq2map is the one used for the kinematic coverage plot, so when a new data with `identity` is being used, it just outputs k1 and k2 so that the program doesn't crash but the results are wrong).